sdk/communication/Azure.Communication.CallAutomation/src/Models/Events/RecognizeCompleted.cs

```diff
@@ -52,7 +81,30 @@ public static RecognizeCompleted Deserialize(string content)
             using var document = JsonDocument.Parse(content);
             JsonElement element = document.RootElement;
 
-            return DeserializeRecognizeCompleted(element);
+            RecognizeCompleted parsedRecognizeCompleted = DeserializeRecognizeCompleted(element);
```

Comment:
why do we need to create the object twice? is not possible to add the below logic within DeserializerRecognizeCompleted?

Reply:
yes, the reviewer require the recognizeComplete event only populate one result. Right now, the default recognizecomplete event populate two results as default behavior: 1. dtmf result 2. choice result. However, at the same time, there should only be one result be populated. That's why we add one more processing on this part.

Reply:
Is the default on an autogenerated class that we cannot modify?

Reply:
> Is the default on an autogenerated class that we cannot modify?

Yes, correct, the event schema make the RecognizeComplete not allowed to modify on serialize/deserialize